pkg/users/account-create-dialog.js

```diff
@@ -116,6 +116,15 @@ function validate_username(username, accounts) {
     return null;
 }
 
+function validate_real_name(real_name) {
+    if (!real_name)
+        return _("No real name specified");
+
+    const real_name_chars = Array.from(real_name);
+    if (!real_name_chars.every(character => character != ':'))
```

Comment:
Heh, that's a rather complicated and slow way of saying ` if (real_name_chars.includes(':'))`

Reply:
Thanks, yeah I noticed that as well, but I thought it was just my computer being funky again.
I am going to change it to use .includes instead of the .every()

I will have a go at the integration test. Never done one in Python, so this will be fun.

Reply:
Please read test/README.md first, that should hopefully get you started (it refers you to HACKING.md for installing the test dependencies). If you have troubles, please yell here or on IRC (libera.chat) in `#cockpit`, that's usually the fastest way to sort this out.